src/region.js

```diff
@@ -288,6 +300,8 @@ _.extend(Region.prototype, CommonMixin, {
     delete this.currentView;
 
     if (!view._isDestroyed) {
+      view._isShown = false;
```

Comment:
this is a very small thing, but for consistency... in collectionview this flag is set after the detach takes place.  Can we move the to right before the `_stopChildViewEvents`?